Habinezajanvier/edified-gen-api
# Use Seeders to Create Admin

### What does this PR do? 
- use seeders to create admin

### How can it be manually tested? 
- Checkout the branch 
- Make sure that you have updated `.env` file. 
- run `npx sequelize-cli db:migrate:undo  --name 20200930033019-create-user` to undo users migration
- run `npx sequelize-cli db:migrate  20200930033019-create-user` to re-create users migration 
- run `npx sequelize-cli db:seed:all` to run seeds 
- Start the server 
- Login using a client app, it should work just fine like before.
